src/pykoala/corrections/sky.py

```diff
@@ -241,6 +243,12 @@ def medfilt_continuum(data, window_size=5):
         continuum = scipy.signal.medfilt(data, window_size)
         return continuum
 
+    @staticmethod
+    def percentile_continuum(data, percentile, window_size):
+        continuum = scipy.ndimage.percentile_filter(data, percentile,
+                                                    window_size)
+        return continuum
```

Comment:
Why this alias?

Reply:
Why not?

Reply:
Let's leave as it is now, and we can revisit this later on.